Cartoon effect

RajS18 opened the pull request:
Here **cartoonize** and **comic** functions implement the cartoon and animated effects on the user provided images resulting in 3 output files (as a result of 1 effect's call). A variety of cartoon effects can be generated for a single input (so as to demonstrate). 
Results are really mesmerizing (by local survey on different images) [added in assets directory].

RajS18 committed:
Cartoon effect has been added, resulting in 3 output images

smaranjitghose commented on cartoon_effects.py:
```diff
@@ -0,0 +1,65 @@
+
+res_img1,res_img2 = cartoonize(img)
+res_img3 = comic(img)
+cv2.imwrite("rajop/cartoon1.jpg", res_img1)
```
As a script intended for general use, I would suggest modifying the output path as 'assets/cartoon_effect.jpg'

RajS18 replied:
On it sir.

smaranjitghose commented on cartoon_effects.py:
```diff
@@ -0,0 +1,65 @@
+import time 
+
+def comic(img):
+    # do edge detection on a grayscale image
```
Nice work with the documentation!

RajS18 replied:
Good to here sir.

smaranjitghose commented:
Also, try to put the comic cartoon effect in a different script

RajS18 committed:
Cartoon Effect - changed the path for saving O/P

Used the default path to "assets" directory to save the output file generated by the code.

RajS18 committed:
Vintage Anime Effect

RajS18 committed:
Merge branch 'master' of https://github.com/RawX18/ArtCV

RajS18 committed:
Vintage Anime Effect

RajS18 committed:
Update anime_effect.py

RajS18 committed:
Cartoon effect and comic cartoon effect

RajS18 committed:
Vintage Anime Effect

RajS18 committed:
anime effect is on separate branch

This is only for Cartoon and comic cartoon effect.

RajS18 committed:
Anime effect results removed.

Anime effect results are needed to be removed from this branch.

RajS18 committed:
Anime effect results removed

Anime effect results are needed to be removed from this branch.

RajS18 committed:
Anime effect results removed.

Anime effect results are needed to be removed from this branch.

RajS18 committed:
Anime effect results removed.

Anime effect results are needed to be removed from this branch.

RajS18 committed:
Merge branch 'master' of https://github.com/RawX18/ArtCV

RajS18 commented:
Greetings Sir,
Please review the changes and if everything is fine merge them, else i'll update accordingly.
Thank you.

smaranjitghose commented on comic_cartoon_effect.py:
```diff
@@ -0,0 +1,44 @@
+img = cv2.imread((args["image"]))
+
+
+print("Wait, Work is in Progess.")
```
You can replace this with progress bars using tqdm. Try it in another PR, once this gets merged

smaranjitghose commented on cartoon_effects.py:
```diff
@@ -0,0 +1,65 @@
+ 
+          
+# Driver's code 
+#img = cv2.imread("assets/kolkata.jpg")
```
Can you please remove this? We need lots of comments but only for explaining our code 😊

RajS18 replied:
Sure Sir.

RajS18 committed:
Update comic_cartoon_effect.py

RajS18 committed:
Update cartoon_effects.py

RajS18 commented:
Greetings Sir.
Update the Python script as per requirement.
Please review the code and if everything's fine please do the merge and accept the PR.

Thank you Sir.
